command/init.go

```diff
@@ -90,6 +102,16 @@ Init Options:
   -key-threshold=3        The number of key shares required to reconstruct
                           the master key.
 
+  -pgp-keys               If provided, must be a comma-separated list of
+                          files on disk containing binary-format public PGP
+                          keys. The number of files must match 'key-shares',
+                          or you can omit 'key-shares' if using this option.
```

Comment:
Also simplifies this. Hard to reason about this. Easier to say must match the shares exactly.